docs/source/1.0/guides/migrating-idl-1-to-2.rst

```diff
@@ -119,6 +119,63 @@ Needs to be updated to:
     }
 
 
+Add the default trait to streaming blobs and strings with the httpPayload trait
+===============================================================================
+
+Blob and string members that are marked with the :ref:`httpPayload-trait` and
+the :ref:`streaming-trait` have always had an implicit default empty value. In
+IDL 2.0, that will become explicit. Any such members that are not already
+marked with the :ref:`required-trait` will now need to be marked with the
+:ref:`default-trait`.
+
+For example, the following model:
+
+.. code-block:: smithy
+
+    $version: "1.0"
+
+    namespace smithy.example
+
+    structure OptionalPayload {
+        // This needs to be updated since it doesn't have the required or
+        // default trait already.
+        @streaming
```

Comment:
This should be on the targeted shape and is [only valid on `blob`](https://awslabs.github.io/smithy/1.0/spec/core/stream-traits.html#streaming-trait). Needs updating in a few places.